locust/contrib/postgres.py

```diff
@@ -0,0 +1,49 @@
+from locust import TaskSet, User, events
+
+import time
+
+import psycopg
+
+
+def create_conn(conn_string):
+    return psycopg.connect(conn_string)
+
+
+def execute_query(conn_string, query):
+    db_conn = create_conn(conn_string)
+    return db_conn.cursor().execute(query)
+
+
+class PostgresClient:
+    def __getattr__(self, name):
```

Comment:
I dont think you need the whole `__getattr__` song and dance :)

Just define the execute_query method directly on the PostgresClient class.